db/migrate/20200323225532_create_reviews.rb

```diff
@@ -0,0 +1,10 @@
+class CreateReviews < ActiveRecord::Migration[5.1]
```

Comment:
Was wondering where this was. Is there a reason it's on this PR and not previous?